src/pages/dashboard/index.vue

```diff
@@ -0,0 +1,311 @@
+                      h-[42px]
+                      rounded-[50%]
+                      bg-shade-gray
+                      border-[2px] border-white
+                    "
+                  />
+                  <Label
+                    class="text-medium-gray mt-[12px]"
+                    :text="$t('portfolio_collected_tab_no_item')"
+                  />
+                </div>
+              </div>
+            </NFTPortfolioCard>
+            <NFTPortfolioItem
+              v-for="id in ownedNFTClassIds"
+              :key="id"
+              :class-id="id"
+            />
+          </ul>
+
+          <ul
+            v-if="currentTab === 'created'"
+            :class="[
+              'w-full',
+              'mx-auto',
+
+              'columns-1',
+              'laptop:columns-2',
+
+              'gap-[16px]',
+            ]"
+          >
+            <NFTPortfolioItem
+              v-for="id in sellingNFTClassIds"
+              :key="id"
+              :class-id="id"
+            />
+          </ul>
+          <div class="flex flex-col items-center my-[48px] w-full">
+            <div class="w-[32px] h-[2px] bg-shade-gray mb-[32px]" />
+            <ButtonV2
+              preset="outline"
+              :text="$t('portfolio_finding_more_button')"
+              to="/campaign/writing-nft"
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+  </Page>
+</template>
+
+<script>
+import { getUserSellNFTClasses, getAddressLikerIdMinApi } from '~/util/api';
+import { getNFTs } from '~/util/nft';
+import { ellipsis, copyToClipboard } from '~/util/ui';
+import { logTrackerEvent } from '~/util/EventLogger';
+
+import walletMixin from '~/mixins/wallet';
+import alertMixin from '~/mixins/alert';
+
+export default {
+  name: 'NFTMyDashboard',
+  layout: 'default',
+  filters: {
+    ellipsis,
+  },
+  mixins: [walletMixin, alertMixin],
+  head() {
+    const name = ellipsis(this.userDisplayName);
+    const title = this.$t('portfolio_title', { name });
+    const description = this.$t('portfolio_description', { name });
+    return {
+      title,
+      meta: [
+        {
+          hid: 'og:title',
+          property: 'og:title',
+          content: title,
+        },
+        {
+          hid: 'description',
+          name: 'description',
+          content: description,
+        },
+        {
+          hid: 'og:description',
+          property: 'og:description',
+          content: description,
+        },
+        {
+          hid: 'og:image',
+          property: 'og:image',
+          content:
+            this.userAvatar ||
+            `https://avatars.dicebear.com/api/identicon/${this.wallet}/600.png`,
+        },
+      ],
+    };
+  },
+  data() {
+    return {
+      wallet: undefined,
+      userInfo: null,
+      ownedNFTs: [],
+      sellingNFTClassIds: [],
+      currentTab: ['collected', 'created'].includes(this.$route.query.tab)
+        ? this.$route.query.tab
+        : 'created',
```

Comment:
Can move the similar logic of MyDashboard && UserPortfolio into `portfolio` mixin?